Demo/DemoTests/MoyaProviderSpec.swift

```diff
@@ -121,12 +130,33 @@ class MoyaProviderSpec: QuickSpec {
                             endDate = NSDate()
                             done()
                         }
+                        return
                     }
 
                     expect{
                         return endDate?.timeIntervalSinceDate(startDate)
                     }.to( beGreaterThanOrEqualTo(NSTimeInterval(2)) )
                 }
+                
+                it("returns canceled error when delayed execution is canceled") {
+                    let closure = { (target: GitHub) -> (Moya.StubbedBehavior) in
+                        return .Delayed(seconds: 2)
+                    }
+                    
+                    let provider = MoyaProvider(endpointsClosure: endpointsClosure, stubResponses: true, stubBehavior: closure)
+                    
+                    var receivedError: NSError?
+                    let target: GitHub = .Zen
+                    waitUntil(timeout: 3) { done in
```

Comment:
Since we're immediately cancelling, is the `timeout: 3` necessary? I think it uses one second as a default, which would be cool to stick with unless there's a reason not to.